README.md

```diff
@@ -1,77 +1,33 @@
-# Contributing
+# README
 
-Thanks for your interest in helping us maintain our course notes! We are happy to consider contributions from faculty and students to help make these notes easier to use and improve their correctness.
+This template repository is set up for you to create and deploy an OER customized for Seneca College ITAS/SDDS. To create a repo based on this template:
 
-## Requirements
+1. Click on the green "Use this template" button near top right corner of code tab
 
-These notes are built using [Docusaurus](https://docusaurus.io/), a modern static website generator. To use it, you'll need to have the following installed:
+2. You will see a page that will let you do some customization. Here are the settings you will want to pay attention to:
 
-- an [LTS version of node.js](https://nodejs.org/en/), which can be checked by running `node -v`. Currently that's node.js 14.x, but confirm with the official site.
+- **Owner** - by default, the owner is set to your account, thus the OER will be created under your github account. If you wish it to be part of the Seneca-ICTOER organization, change this from the drop down menu
+- **Repository Name** - If the OER is used by just one course, the course code can be a good name for this repository. If the OER is used by multiple courses, it is best to use a descriptive name that is applicable for all courses that will use the OER
+- **Public vs Private** - Your repository can be private but when notes are deployed, they will become public, the notes website will be public regardless of the privacy setting for this repository. Whether the repo is public or private can be changed later so if you are not sure you can start with repo being private and make the change once your repository is in a state where you wish to share it.
+- **Include all branches** checkbox. It is recommended that this remains unchecked as it is unlikely that the other branches of this template repository will be useful to you.
 
-- The [yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable) package manager, version >= 1.5 (which can be checked by running `yarn --version`).
+This is a screenshot of the recommended set of settings for an initial repository:
 
-- [Visual Studio Code](https://code.visualstudio.com/). This is not strictly necessary, but we encourage it since we use various extensions to make working on the site easier.
+![](static/img/templatesettings.png)
 
-## Installation
+Once you are happy with your settings, hit the Create repository from template button.
```

Comment:
Grammar:  ...from ***`the`*** template button

Reply:
This is the writing on the actual button so we shouldn't add extra words.  I've quoted it instead.